tests/API/Helpers/State/DummyStateHandlerTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Auth0\Tests\Api\Helpers\State;
+
+use Auth0\SDK\API\Helpers\State\DummyStateHandler;
+
+/**
+ * Class DummyStateHandlerTest
+ *
+ * @package Auth0\Tests\Api\Helpers\State
+ */
+class DummyStateHandlerTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * State handler to use.
+     *
+     * @var DummyStateHandler
+     */
+    private $state;
+
+    /**
+     * DummyStateHandlerTest constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+        $this->state = new DummyStateHandler();
+    }
+
+    /**
+     * Test that the state issued is null.
+     */
+    public function testStateIssuedCorrectly()
+    {
+        $this->assertNull($this->state->issue());
```

Comment:
how is a "null" state correct?

Reply:
This is a dummy class so `null` is desired in this case.

Reply:
Yes :)

Reply:
Yes ;)